src/lib/integration_test_lib/network_state.ml

```diff
@@ -28,6 +28,10 @@ module Make
     ; global_slot : int
     ; snarked_ledgers_generated : int
     ; blocks_generated : int
+    ; persisted_frontier_loaded : int
+    ; persisted_frontier_fresh_boot : int
+    ; bootstrap_required : int
+    ; persisted_frontier_dropped : int
```

Comment:
this is a bit of a nitpick but all of these variable names sound like they should be booleans with true/false, like  "persisted_frontier_loaded", however instead they are ints.  can you rename them something better?  maybe "num_times_persisted_frontier_loaded" or something like that, if you can find something terser that'd be good.